python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -369,6 +369,8 @@ def ast_to_func(ast_root, dyfunc, delete_on_exit=True):
     function, the other inner functions are invisible for the decorated function.
     """
     source = ast_to_source_code(ast_root)
+    import_fluid = "import paddle.fluid as fluid\n"
```

Comment:
If you have time, can we put these at two different function?

ast_to_func means ast_to_func. When you add these two lines, you mean ast_to_func for paddle. The the function cannot be re-used if we re-use the function for non-paddle thing at the future.

One way is:

def new_func(...):
       source = ast_to_source_code(ast_root)
       source = import_fluid + source
       return source_to_func(source)

But your change also `fine` to me (In English, fine is not as good as `good`). If you don't have time, tell me and I can also approve this PR.

Reply:
After discuss offline, there are many places call ast_to_func and I’m fine with current change. I will approve

Reply:
OK. It is a good suggestion. Thanks.